### PR TITLE
Make file changes to fix #issue 496

### LINKED ISSUE #496: build :  cross compilation of edge_core binary on arm arch fails.
**What happened**:
When we make edge_core(inside edge folder),cross compilation of edge_core is not sucessfull for arm architecture.
**What you expected to happen**:
file edge_core(command must show you ARM,in this case its not showing)
**How to reproduce it (as minimally and precisely as possible)**:
cd $GOPATH/src/github.com/kubeedge/kubeedge/edge
make edge_core
file edge_core(check arm is there in the console output)
**Anything else we need to know?**:

**Environment**:
- KubeEdge version:
- Hardware configuration:
- OS (e.g. from /etc/os-release):
- Kernel (e.g. `uname -a`):
- Others:

### PR DESCRIPTION
**Pull request by srivatsav123**
**What type of PR is this?**
/kind bug

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #496  

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by srivatsav123** [comment]
Result after fix
armsupport

File: edge/Makefile [hunk 1]
```diff
@@ -31,13 +31,12 @@ clean:
...
 cross_build:
-	export GOARCH=arm
-	export GOOS="linux"
-	export GOARM=6
-	export CGO_ENABLED=1
-	export CC=arm-linux-gnueabi-gcc
-	go build cmd/edge_core.go
-
+	export GOARCH=arm \
+        export GOOS="linux" \
```
Comment (rohitsardesai83) [inline edge/Makefile]: please check the indentation.

Comment (sids-b) [reply edge/Makefile]: +1... Rest looks good to me

Comment (srivatsav123) [reply edge/Makefile]: i checked its correct

**Pull request merged**